### ISSUE TITLE
kfctl generate fails "jupyter" is not a component or a module

### ISSUE THREAD

**Issue by jlewi** [kubeflow/kubeflow#2796]
I started seeing the following failures in the presubmit to #2795.

My PR is using: 4addac44f235fa89145e586be5dcb7fbd94fe261

```
util.py                     71 INFO     time="2019-03-24T21:53:18Z" level=info msg="Writing component at '/mnt/test-data-volume/jlewi-kfctl-test-2795-0324
-143944/kfctl-3944/ks_app/components/tf-job-operator.jsonnet'" filename="component/create.go:92"
util.py                     71 INFO     Error: couldn't generate KfApp: coordinator Generate failed for ksonnet: Failed to set param jupyter platform gke:
 could not find component: extracting module and component names from jupyter: "jupyter" is not a component or a module
```

My suspicion is this is related to #2764 which removed the jupyter component from one of the config files.

I suspect there are some versioning issues here; e.g. when we call `kfctl generate` in the test we aren't pulling the config from the release branch so we aren't testing config changes in the presubmit.

/cc @zabbasi @gabrielwen @kkasravi

**Comment by jlewi** [comment]
It looks like we no longer define the jupyter component
https://github.com/kubeflow/kubeflow/blob/3fa97bc61e7b4b1eeb0062a8cbc98920e9da43d5/bootstrap/config/kfctl_basic_auth.yaml#L24

But we are still trying to set parameters on it.
https://github.com/kubeflow/kubeflow/blob/3fa97bc61e7b4b1eeb0062a8cbc98920e9da43d5/bootstrap/config/kfctl_basic_auth.yaml#L66

**Comment by jlewi** [comment]
Working on a fix in #2795.
